subject: Change package name to microG variant

Append .microg so this installs alongside the stock build. microG grants signature
spoofing per package, so after the rename it must be re-enabled in microG settings.
---
--- a/chrome/android/chrome_public_apk_tmpl.gni
+++ b/chrome/android/chrome_public_apk_tmpl.gni
@@ -29,6 +29,7 @@ declare_args() {
   if (use_stable_package_name_for_trichrome) {
     chrome_public_manifest_package += ".stable"
   } else if (android_channel != "default") {
     chrome_public_manifest_package += "." + android_channel
   }
+  chrome_public_manifest_package += ".microg"
 }
 
 chrome_public_test_manifest_package = "org.ungoogled.chromium.tests"
--- a/chrome/android/java/res_chromium_base/values/channel_constants.xml
+++ b/chrome/android/java/res_chromium_base/values/channel_constants.xml
@@ -7,7 +7,7 @@
 
 <resources xmlns:android="http://schemas.android.com/apk/res/android">
     <!-- The application name displayed to the user. -->
-    <string name="app_name" translatable="false">Ungoogled Chromium</string>
+    <string name="app_name" translatable="false">Ungoogled Chromium microG</string>
     <string name="bookmark_widget_title" translatable="false">Chromium bookmarks</string>
     <string name="search_widget_title" translatable="false">Chromium search</string>
     <string name="quick_action_search_widget_title" translatable="false">Chromium quick action search</string>
